src/Variables/SingleChoiceVariable.php

```diff
@@ -26,7 +26,7 @@ class SingleChoiceVariable implements ClosedVariableInterface
     /**
      * @param string $name
      * @param array<string, string> $titles
-     * @phpstan-param non-empty-list<ValueOptionInterface> $valueOptions
+     * @phpstan-param array<ValueOptionInterface> $valueOptions
```

Comment:
This is not a keyed array; so if it really can not be `non-empty-list` then it should be `list`. Why was the old annotation wrong?